python/paddle/v2/fluid/optimizer.py

```diff
@@ -54,14 +82,20 @@ def _create_param_lr(self, param_and_grad):
         param_lr = param.optimize_attr['learning_rate']
         param_lr_shape = [1]
         param_lr_var = self.helper.create_global_variable(
-            name=unique_name("learning_rate"),
+            name=unique_name("learning_rate_" + param.name),
             dtype='float32',
             shape=param_lr_shape,
             lod_level=1,
             persistable=True)
-        param_lr = param_lr * self._learning_rate
+        param.block.program.global_block().append_op(
+            type="scale",
+            inputs={"X": self._global_lr_var},
+            outputs={"Out": param_lr_var},
+            attrs={"scale": param_lr})
+        # give param learning rate a default value 0.0
```

Comment:
What's the purpose of default value 0.0?

Reply:
removed